translator/toenvconfig/toEnvConfig.go

```diff
@@ -16,8 +16,9 @@ import (
 )
 
 const (
-	userAgentKey = "user_agent"
-	debugKey     = "debug"
+	userAgentKey      = "user_agent"
+	debugKey          = "debug"
+	awsSDKLogLevelKey = "aws_sdk_log_level"
```

Comment:
Minor: I think the casing should be `awsSdkLogLevelKey`.

Reply:
Did you see this comment?

Reply:
Fixed.